modules/generator/processor/servicegraphs/servicegraphs.go

```diff
@@ -294,12 +294,21 @@ func (p *Processor) upsertDatabaseRequest(e *store.Edge, resourceAttr []*v1_comm
 
 	// Check for db.system only if we don't have db.name above
 	if !isDatabase {
-		if _, ok := processor_util.FindAttributeValue(string(semconv.DBSystemKey), resourceAttr, span.Attributes); ok {
+		if name, ok := processor_util.FindAttributeValue(string(semconv.DBSystemKey), resourceAttr, span.Attributes); ok {
```

Comment:
The `semconv` package at the top of this file does not support the referenced key.  We will need to update this in order for this to work.  I believe this also warrants a changelog entry.